sections/slideshow.liquid

```diff
@@ -114,7 +114,7 @@
           {%- endif -%}
         </div>
         <div class="slideshow__text-wrapper banner__content banner__content--{{ block.settings.box_align }} page-width{% if block.settings.show_text_box == false %} banner--desktop-transparent{% endif %}">
-          <div class="slideshow__text banner__box content-container content-container--full-width-mobile color-{{ block.settings.color_scheme }} gradient slideshow__text--{{ block.settings.text_alignment }} slideshow__text-mobile--{{ block.settings.text_alignment_mobile }}">
+          <div class="slideshow__text banner__box {% if block.settings.heading != blank or block.settings.subheading != blank or block.settings.button_label != blank %}banner__box-padding {% endif %}content-container content-container--full-width-mobile color-{{ block.settings.color_scheme }} gradient slideshow__text--{{ block.settings.text_alignment }} slideshow__text-mobile--{{ block.settings.text_alignment_mobile }}">
```

Comment:
In the context of the Slideshow section, taking a look at the current block level setting for the Image slide, **Show container on desktop**. If no textual content exists, the container will still display but we also have the option of unchecking the "Show container on desktop" setting to hide this empty, blank container:

<img src="https://user-images.githubusercontent.com/28404165/178055927-64319f62-fa16-4e46-b620-757b2ed474b4.png" width="500" height="auto">

I wonder if it would make more sense to add a new setting of **Show container on mobile** to the Slideshow section. This would remove the blank container that appears when no textual content exists and we'd only need to add one Liquid if condition vs. multiple (less code):

<img src="https://user-images.githubusercontent.com/28404165/178056196-6dc81cd7-351a-4af6-baf4-dee04c26a58f.png" width="500" height="auto">

This pattern / setting already exists in the **Image banner** section:

<img src="https://user-images.githubusercontent.com/28404165/178056879-f520768f-349a-4e63-8560-c11b9af11e0d.png" width="500" height="auto">

And lastly, for parity, we should also consider adding whatever we decide in this PR to the **Email signup banner** section:

<img src="https://user-images.githubusercontent.com/28404165/178057415-757728af-a254-4f45-91e1-c39c2fb3696f.png" width="500" height="auto">